front/src/Data.elm

```diff
@@ -13,12 +13,28 @@ type IngredientType
 
 
 type Glass
-    = OldFashioned
-    | Collins
+    = -- Tumblers:
+      Collins
     | Highball
+    | OldFashioned
+    -- | Shot
+    -- | Table
+    -- Stemware:
+    -- | Absinthe
+    -- | Chalice
+    -- | ChampagneCoupe
     | ChampagneFlute
-    | CopperMug
     | Cocktail
+    | Hurricane
+    | Margarita
+    -- | Sherry
+    -- | Snifter
+    | Wine
+    -- Other:
+    | CopperMug
+    | IrishCoffeeMug
+    | SteelCup
+    | ZombieGlass
```

Comment:
Used the naming from Wikipedia‘s glassware.

Reply:
TIL that an Irish cofee mug exists :)